### PR TITLE
Added yellow message bar on review screen

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#13957
yellowMessage

*Description of changes:*
-Made intro text dependent on review mode or not
-Added yellow message bar on review screen
-Added test for this message

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by cameron-eyds — APPROVED** [review]
Looks goood 👍

File: ppr-ui/tests/unit/MhrInformation.spec.ts [hunk 1]
```diff
@@ -394,4 +394,27 @@ describe('Mhr Information', () => {
...
+
+    // trigger review
+    wrapper.find('#btn-stacked-submit').trigger('click')
+    await Vue.nextTick()
+
+    // exists on review page
+    expect(wrapper.find('#yellow-message-bar').exists()).toBeTruthy()
+
+    //trigger back button
+    wrapper.find('#btn-stacked-back').trigger('click')
```
Comment (dimak1) [inline ppr-ui/tests/unit/MhrInformation.spec.ts:452]: Good idea to test the bar on the previous screen 👍🏻

**Review by dimak1 — APPROVED** [review]

**Comment by cameron-eyds** [comment]
> Looks goood 👍

It looks like the bottom border is missing though, maybe its just the screen shot?

File: ppr-ui/tests/unit/MhrInformation.spec.ts (hunk 1 above)
Comment (owillborn) [reply ppr-ui/tests/unit/MhrInformation.spec.ts:452]: thanks!

**Comment by owillborn** [comment]
> > Looks goood 👍
> 
> It looks like the bottom border is missing though, maybe its just the screen shot?

@cameron-freshworks Yea, just the screenshot.. table displays properly

**Review by ChasBoucher — APPROVED** [review]
Looks good...

**Review by doug-lovett — APPROVED** [review]

**Pull request merged**